doc/doxygen/src/css/riot.css

```diff
@@ -129,7 +131,8 @@ div.toc-sm ul {
   margin: 0;
   padding-left: 10px;
 }
-div.toc li, div.toc-sm li {
+div.toc li,
+div.toc-sm li {
```

Comment:
I hand-edited these changes and applied the change only after the fact to the LESS file. The auto-generated code looks a little bit different though ;-)